src/main/ipc/installer.ts

```diff
@@ -98,11 +85,44 @@ async function github(
   };
 }
 
+const STORE_BASE_URL = `https://replugged.dev/api/v1/store`;
```

Comment:
this should try to use the backend URL from settings that a user can specify

Reply:
@12944qwerty we do not have an option for they currently so that's out of scope for this PR.